### PR TITLE
added new message for firsttimeInDashboard

### PR DESCRIPTION
**Pull request by Josmorsot**

### CODE REVIEW

**Comment by padawannn** [comment]
Change this PR against connectors

File: lib/assets/javascripts/new-dashboard/pages/Home/ConnectionsSection/ConnectionsSection.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
```
Comment (padawannn) [inline lib/assets/javascripts/new-dashboard/pages/Home/ConnectionsSection/ConnectionsSection.vue:1]: unlike datasetSection and MapSection, I think that ConnectionsSection component is not necessaryy

File: lib/assets/javascripts/new-dashboard/pages/Data/Connections/Connections.vue [hunk 2]
```diff
@@ -79,6 +98,9 @@ export default {
     this.$store.dispatch('connectors/fetchConnectionsList');
   },
   methods: {
+    createConnection () {
+      this.$emit('newConnectionClicked');
```
Comment (padawannn) [inline lib/assets/javascripts/new-dashboard/pages/Data/Connections/Connections.vue:102]: It's not necessary to emit a  newConnectionClicked event, instead replace this `<button @click="createConnection" class="button is-primary">{{ $t(`DataPage.zeroCase.createDataset`) }}</button>` by 
`<router-link :to="{ name: 'new-connection' }" v-if="connections.length">
              <button class="button is-primary">{{ $t(`DataPage.newConnection`) }}</button>
            </router-link>`

**Review by padawannn — CHANGES_REQUESTED** [review]

**Review by padawannn — APPROVED** [review]

**Comment by rafatower** [comment]
:robot: please make sure all the checks pass before merging @Josmorsot @padawannn

**Pull request merged**